packages/ui/Skinnable/useSkins.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import {determineSkin, determineVariants, getClassName} from './util';
+
+/**
+ * Allows a component to respond to skin changes via the Context API
+ *
+ * Example:
+ * ```
+ * <App skin="dark">
+ * 	<Section>
+ * 		<Button>Gray Button</Button>
+ * 	<Section>
+ * 	<Popup skin="light">
+ * 		<Button>White Button</Button>
+ * 	</Popup>
+ * </App>
+ * ```
+ *
+ * @class SkinContext
+ * @memberof ui/Skinnable
+ * @hoc
```

Comment:
Is this really a hoc?  Is it a class?  I think we need a different thing for this.

Reply:
It's private so I just removed the pragma.